backend/core/src/shared/email/email.service.ts

```diff
@@ -177,11 +177,11 @@ export class EmailService {
       false,
       (error) => {
         if (error instanceof ResponseError) {
-          const { response } = error
-          const { body } = response
-          console.error(`Error sending email to: ${to}! Error body: ${body}`)
+          console.error(
```

Comment:
I think this might have been fixed already upstream and is part of the upstream merge?

https://github.com/CityOfDetroit/bloom/pull/717/files#diff-dc47a8171d630fffa456ddc9d99cd2ac7575a65c8562ef0fb7c33086063b1390

Reply:
Ah, so it is! Thanks, should have checked first.